.github/workflows/publish-next.yml

```diff
@@ -72,6 +72,18 @@ jobs:
         env:
           ENDPOINT: ${{ secrets.LIVENESS_BETA_SAMPLE_APP_BUILD_TRIGGER }}
 
+  build-test-v2:
+    uses: ./.github/workflows/reusable-build-system-test.yml
+    needs: publish
+    with:
+      dist-tag: next
```

Comment:
Just for my own understanding, is there any logic required to ensure that the just published `next` version is available on NPM and that we aren't installing a more or less "stale" `next` version?

Reply:
We don't proceed with the tests if `yarn publish` fails and from their documentation it seems like if it succeeds then the package should immediately be available. https://classic.yarnpkg.com/lang/en/docs/publishing-a-package/

We could also compare the version we generate with the currently available version to be sure.